src/tokenizer/stop_word_filter.rs

```diff
@@ -98,3 +98,15 @@ where
         false
     }
 }
+
+impl Default for StopWordFilter {
+    fn default() -> StopWordFilter {
```

Comment:
Maybe have a specific function called `.english()` and have `.default()` call that one.

Reply:
got it